src/cmdlib.sh

```diff
@@ -417,15 +426,19 @@ EOF
         done
     fi
 
-    # Store the fully rendered disk image config (image.json) inside
+
+    # Store the fully rendered disk image config (image.json)
+    # and the platform (platforms.json) inside
     # the ostree commit, so it can later be extracted by disk image
     # builds.
-    local imagejsondir="${tmp_overridesdir}/imagejson"
+    local jsondir="${tmp_overridesdir}/jsons"
     export ostree_image_json="/usr/share/coreos-assembler/image.json"
-    mkdir -p "${imagejsondir}/usr/share/coreos-assembler/"
-    cp "${image_json}" "${imagejsondir}${ostree_image_json}"
-    commit_overlay cosa-image-json "${imagejsondir}"
-    layers="${layers} overlay/cosa-image-json"
+    export ostree_platforms_json="/usr/share/coreos-assembler/platforms.json"
+    mkdir -p "${jsondir}/usr/share/coreos-assembler/"
+    cp "${image_json}" "${jsondir}${ostree_image_json}"
+    cp "${platforms_json}" "${jsondir}${ostree_platforms_json}"
+    commit_overlay cosa-json "${jsondir}"
+    layers="${layers} overlay/cosa-json"
```

Comment:
could this be simplified down into the following or maybe I'm missing something:

```
    local jsondir="${tmp_overridesdir}/jsons"                                      
    mkdir -p "${jsondir}/usr/share/coreos-assembler/"                              
    cp "${image_json}" "${platforms_json}" "${jsondir}/usr/share/coreos-assembler/"
    commit_overlay cosa-json "${jsondir}"                                          
    layers="${layers} overlay/cosa-json"                                           

```

Reply:
Well, it appears that we might not actually need to export it. I couldn't find any other part of the code using these variables outside of this specific section. What's interesting is that almost all the variables in this file are being exported, but it seems like we can keep these ones local to this script.